.github/workflows/aws-heroku-helper.sh

```diff
@@ -19,6 +19,8 @@ function build_editor() {
   cd "./moped-editor";
   print_header "Building the editor";
   npm install && npm run build:development;
+  print_header "Clearing Bucket";
+  aws s3 rm s3://atd-moped-editor-development --recursive;
```

Comment:
out of curiosity, what was happening f we didnt clear the bucket?

Reply:
It was a precaution step, I was not seeing some changes I had made t the front end. @chiaberry

Reply:
didnt know if it was a nice to have or just in case, or if there were side effects happening

thanks for always answering my questions